schemas/chambers/index.ts

```diff
@@ -0,0 +1,20 @@
+import { FeatureSchemaDefinition } from "schemas/types";
+
+export const chambers: FeatureSchemaDefinition = {
+  variants: {
+    card: [
+      {
+        title:
+          "[Chambers] Left Vertical Media With Bullet Line and Right Content",
```

Comment:
Can't have any other nomenclature. Which should be more generic.

Reply:
can it be somthing like "vertical oriented Left media", "horizonatal oriented Left media"

Reply:
Make it specific.